Karma-Khameleons/bangazon-website
# Wc product detail

## Description
Add description about what this pull request covers.
* added: 
- product_detail_view.py(view)
- product_detail.html(template) 
- test_product_detail.py(test)
- url for product_detail

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#13 

## Problem to Solve
Describe the problem(s) this pull request solves.
we now have an interface for customers to view product details when they click on a product

## Expected Behavior
Describe the expected behavior.
* test for view should pass
* localhost/product_detail - should show the interface for listing product detail

## Steps to Test Solution
List steps to test your solution
```
git fetch
git checkout wc-productDetail
python manage.py test bang_app
```

part 2:
run server & go to '/product_detail' to make sure template is there 

## Unit Testing

## Documentation


## Deploy Notes

Notes regarding deployment the contained body of work. These should note any
db migrations, etc.
